CRM/Upgrade/Incremental/php/FiveZero.php

```diff
@@ -142,6 +161,26 @@ public function upgrade_5_0_2($rev) {
     }
   }
 
+  /**
+   * Updates current case types so they have a category assigned. All case types
+   * are assigned the Workflow category by default except for the Application case
+   * type, which gets the Vacancy category.
+   */
+  protected function _setDefaultCategoriesForCaseTypes() {
+    $caseTypes = civicrm_api3('CaseType', 'get', [
+      'options' => [ 'limit' => 0 ]
+    ]);
+
+    foreach ($caseTypes['values'] as $caseType) {
+      $category = $caseType['name'] === 'Application' ? 'VACANCY' : 'WORKFLOW';
```

Comment:
The value of the category property of the case type is actually supposed to be the value property of the option values and not the name of the option value.

Reply:
You can pass the name of the pseoudo consant to the API and it will automatically turn them into their corresponding value (I double checked that).

I have done it this way since I don't have the value at hand, I'd need to query the DB for it or set it myself. But setting the value manually does not seem to be a core convention:

https://github.com/compucorp/civicrm-core/blob/8f9a245ba678dcfe2a023f85060d065989c1926d/CRM/Upgrade/Incremental/php/FourSeven.php#L904-L918